examples/slide.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>Asciidoctor in JavaScript powered by Opal</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700">
+  <link rel="stylesheet" href="asciidoctor.css">
+  <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/1.8.3/jquery.js"></script>
+  <style>
+    .left {
+      width: 50%;
+      float: left
+    }
+
+    .right {
+      width: 50%;
+      float: right
+    }
+
+    iframe, textarea {
+      width: 99%;
+      height: 500px;
+    }
+  </style>
+</head>
+<body>
+<div class="left">
+  <textarea id="editor" cols="30" rows="10">
+= Title
+:backend: revealjs
+:templatedir: slide-templates
+
+== Slide 1
+
+Content 1
+
+== Slide 2
+
+Content 2
+
+  </textarea>
+</div>
+<div class="right">
+  <iframe id="output"></iframe>
+</div>
+<button id="convertBtn">Convert</button>
+<script src="jade.js"></script>
+<script src="../bower_components/opal/opal/current/opal.js"></script>
+<script src="asciidoctor-core.js"></script>
+<script src="asciidoctor-factory.js"></script>
+<script src="asciidoctor-slide.js"></script>
+<script src="asciidoctor-deckjs.js"></script>
+<script src="asciidoctor-revealjs.js"></script>
+<script>
+  var options = Opal.hash2(['safe', 'attributes'], {
```

Comment:
Remember we don't need Opal.hash2 anymore. We can just use Opal.hash and wrap a JSON object.